Assets/AppCenter/Plugins/AppCenterSDK/Core/Shared/AppCenter.cs

```diff
@@ -28,6 +28,16 @@ public static LogLevel LogLevel
             set { AppCenterInternal.SetLogLevel((int)value); }
         }
 
+        /// <summary>
+        /// Destroys static references
+        /// </summary>
+        public static void Cleanup()
+        {
```

Comment:
Probably make sense to add the same changes to `Blank/CrashesDelegate.cs` file too?

Reply:
Done.